scripts/ci/tools/free-space.py

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""free-space.py for cean environment before to start CI"""
+
+import subprocess
+
```

Comment:
Executable scripts should verify that they are running correctly by checking the contents of the `__name__` variable.
We should add the following code at the beginning of the file
```python
if __name__ != "__main__":
    raise SystemExit(
        "This file is intended to be executed as an executable program. You cannot use it as a module."
        f"To run this script, run the ./{__file__} command"
    )
```
or:
Put all the content in the function and add the code at the bottom of the file:
```python
if __name__ == '__main__':
   main()
```

Reply:
Thank you, I am considering:

`if __name__ == '__main__':
    main()`